templates/view_form.tpl.html

```diff
@@ -80,6 +80,8 @@
                             (<a href="#customer_details">{t}Complete Details{/t}</a>)
                         {elseif $row.field == 'percentage_complete'}
                             <div class="ui-progressbar iss_percent_complete" data-percent="{$row.percent}"><div class="progress-label">&nbsp;{$row.percent} %</div></div>
+                        {elseif $row.field == 'priority'}
+							{if $row.pri_icon > 0}<img src="{$core.rel_url}images/priority/{$row.pri_icon|escape:"html"}.png" border="0" align="absmiddle"> {/if}{$row.pri_title|escape:"html"}
```

Comment:
indenting broken. also try to use css not hardcode urls into html.

also, can `pri_icon` value be `0` ?